serverless.yml

```diff
@@ -5,7 +5,7 @@ provider:
   region: ${opt:region, 'us-west-2'}
   stage: ${opt:stage, 'TEST'}
   runtime: java11
-  memorySize: 768
+  memorySize: 256
```

Comment:
Looking at this function's recent invocations, I'm wondering if `256` might be too small. I'm willing to see how 256 shakes out before changing it though.